diff --git a/apps/web/.next/standalone/node_modules/next/dist/server/image-optimizer.js b/apps/web/.next/standalone/node_modules/next/dist/server/image-optimizer.js
--- a/apps/web/.next/standalone/node_modules/next/dist/server/image-optimizer.js
+++ b/apps/web/.next/standalone/node_modules/next/dist/server/image-optimizer.js
@@ -684,6 +684,7 @@ class ImageOptimizerCache {
         ]);
     }
     constructor({ distDir, nextConfig, cacheHandler }){
-        this.cacheDir = (0, _path.join)(/* turbopackIgnore: true */ distDir, 'cache', 'images');
+        const cacheDir = process.env['NEXT_CACHE_DIR'] || (0, _path.join)(/* turbopackIgnore: true */ distDir, 'cache');
+        this.cacheDir = (0, _path.join)(/* turbopackIgnore: true */ cacheDir, 'images');
         this.nextConfig = nextConfig;
         this.cacheHandler = cacheHandler;
